### PR TITLE
Small cleanup and comment

### PR DESCRIPTION
**Pull request by codereport**
Unnecessary use of `std::abs` after calculating `min` and `max` and really we should be using `std::minmax` but until C++17 when we have structured bindings it doesn't improve things much - so just left a comment.

### CODE REVIEW

File: cpp/tests/utilities/timestamp_utilities.cuh [hunk 1]
```diff
@@ -60,9 +60,10 @@ generate_timestamps(int32_t count, time_point_ms start, time_point_ms stop) {
                  .time_since_epoch()
                  .count();
 
+  // When C++17, auto [min, max] = std::minmax(lhs, rhs)
   auto min = std::min(lhs, rhs);
   auto max = std::max(lhs, rhs);
-  auto range = static_cast<Rep>(std::abs(max - min));
+  auto range = static_cast<Rep>(max - min);
```
Comment (harrism) [inline cpp/tests/utilities/timestamp_utilities.cuh]: Since start can be larger than stop, you need the abs to guarantee a positive range.

Comment (codereport) [reply cpp/tests/utilities/timestamp_utilities.cuh]: Isn't that what the `std::min` and `std::max` calls with `lhs` and `rhs` take care of?

Comment (harrism) [reply cpp/tests/utilities/timestamp_utilities.cuh]: Ah yes. Right. But in general this is a nitpick PR that doesn't fix a correctness or performance problem. The review discussion cost is higher than any benefit. Suggest incorporating changes like this along with other changes.

Comment (codereport) [reply cpp/tests/utilities/timestamp_utilities.cuh]: Good point, will do in the future

**Pull request merged**